tests/integration_tests/style/test_rust.py

```diff
@@ -7,6 +7,19 @@
 from framework import utils
 
 
+def test_rust_order(monkeypatch):
+    """
+    Tests that `Cargo.toml` dependencies are alphabetically ordered.
+
+    @type: style
```

Comment:
nit: This is not needed anymore. These @type tags will be removed soon.

Reply:
I think it would make the most sense to have 1 commit/PR that removes them all that can document this.